### ISSUE TITLE
Recent KP.2.3 sublineage with S:T22N (57, 3 countries)

### ISSUE THREAD

**Issue by FedeGueli** [sars-cov-2-variants/lineage-proposals#1789]
Just spotted a new S:T22N sublineage this time on KP.2.3 backbone.
From collection dates it should n ot be fast but worth tracking

KP.2.3> C28714T > C29077T > S:T22N (C21627A)  
Query:C28714T ,C29077T ,C21627A, 
Samples: 7 
Countries: USA (2, NY), France, Canada , Portugal (3 via GBW)

Tree:
<img width="1159" alt="Screenshot 2024-07-28 alle 08 47 38" src="https://github.com/user-attachments/assets/f32f208a-06ba-429e-b018-02d7ffdee58f">
https://nextstrain.org/fetch/genome-test.gi.ucsc.edu/trash/ct/subtreeAuspice1_genome_test_5675a_5e7260.json?label=id:node_7043279

**Comment by FedeGueli** [comment]
8  now, cc @corneliusroemer this is the biggest 22N+31del but as you can see it doesnt seem too fit

**Comment by FedeGueli** [comment]
Apparently slow

**Comment by FedeGueli** [comment]
13

**Comment by FedeGueli** [comment]
+7 from 4 different American States mainly East Coast

**Comment by FedeGueli** [comment]
32 still mostly north America

**Comment by FedeGueli** [comment]
45 for a cluster in NY

**Comment by FedeGueli** [comment]
48, this is doing well right now and it is not clusters

**Comment by Over-There-Is** [comment]
55

**Comment by FedeGueli** [comment]
> 48, this is doing well right now and it is not clusters

i was worng it is in a NY cluster, likely i looked at results on Gisaid reverted from the first,

**Comment by xz-keg** [comment]
+1 GBW from France

**Comment by FedeGueli** [comment]
> +1 GBW from France

thx reopening it

**Comment by FedeGueli** [comment]
one sample only from October (2 GBW from France)

**Comment by FedeGueli** [comment]
> 68 now. Not dead

yeah not dead but dying to me 5 samples in October from high sequenced areas so i don't think it is growing

**Comment by FedeGueli** [comment]
> better

yeah for sure 22N became a key all of a sudden
<img width="998" alt="Screenshot 2024-11-12 alle 06 18 44" src="https://github.com/user-attachments/assets/7d8b53b3-7a5d-4f7d-8875-5e589490a3b7">

**Comment by FedeGueli** [comment]
only 5 not old so closing it
